examples/text-generation/README.md

```diff
@@ -202,6 +202,20 @@ python ../gaudi_spawn.py --use_deepspeed --world_size 8 run_generation.py \
 --flash_attention_causal_mask
 ```
 
+To run Deepseek-R1-BF16 inference on 16 Gaudi3 cards use the following command and replace the hostfile parameter to the correct file:
+```bash
+python3 ../gaudi_spawn.py --hostfile=<hostfile> --use_deepspeed \ 
```

Comment:
can you give an example of hostfile link somewhere?

Reply:
Maybe https://github.com/huggingface/optimum-habana/blob/main/examples/multi-node-training/hostfile ?
Even though this ones is for 4 nodes